### PR TITLE
Add zarr as a fix for zar.

### PR DESCRIPTION
**Pull request by yarikoptic**
Zarr is a new community driven data format, discover more https://en.wikipedia.org/wiki/Zarr_(data_format)
and I ran into this while fixing a Zarr related project

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -63596,8 +63596,8 @@ yuo->you
 yuor->your
 yur->your
 yymmetrical->symmetrical
-zar->czar
-zars->czars
+zar->czar, zarr
+zars->czars, zarrs
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:63600]: `Zarr` is OK, but not sure about the plural `Zarrs`.

Don't forget the trailing comma.

Comment (yarikoptic) [reply codespell_lib/data/dictionary.txt:63600]: sorry, what trailing comma?

Comment (yarikoptic) [reply codespell_lib/data/dictionary.txt:63600]: Let's remove

```suggestion
```

Comment (yarikoptic) [reply codespell_lib/data/dictionary.txt:63600]: I guess not to leave it behind... ok let's do:

```suggestion
zars->czars
```

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**